src/main/java/org/cactoos/collection/Mapped.java

```diff
@@ -38,13 +36,6 @@
  * @param <Y> Type of target item
  * @since 0.14
  */
-@SuppressWarnings
-    (
-        {
-            "PMD.TooManyMethods", "PMD.CyclomaticComplexity",
-            "PMD.StdCyclomaticComplexity", "PMD.ModifiedCyclomaticComplexity"
-        }
-    )
 public final class Mapped<X, Y> extends CollectionEnvelope<Y> {
 
     /**
```

Comment:
`new IterableOf<X>` can be replaced with inferred `new IterableOf<>`

Reply:
@ilyakharlamov there is no `new IterableOf<X>` in this file